packages/mesos-modules/build

```diff
@@ -1,25 +1,25 @@
 #!/bin/bash
 
-pushd /pkg/src/mesos-modules/
-./bootstrap
-popd
-
+# TODO(akornatskyy): protoc is compiled with shared libaries (a part of mesos
+# dcos package) and needs LD_LIBARY_PATH to be set correctly, this is done by
+# sourcing the `environment.export` file below.
```

Comment:
I might recommend removing the TODO from this comment, but leave a comment which explains why we need to source the environment file.

Reply:
done